SLO Dashboard Log Search functionality (#1582)
SumoLogic/sumologic-documentation: docs/observability/reliability-management-slo/dashboards.md

</table>

## Setting SLO Data Granularity

### Time Ranges

To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods.<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />

### Compliance Period

You can also filter by compliance period to view your past activity and plan ahead.<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />

### Open in Log Search

You can launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down into further granular details.
1. Go to **Manage Data** > **Monitoring** > **SLO**.

> 1. Click on any SLO line item.
> 1. Hover over the panel, click the three-dot kebab icon, and select the **Open in Log Search** option.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>

Author: ```suggestion
1. Double-click on any SLO line item.
1. Hover over the panel > Click the kebab icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
```

</table>

## Setting SLO Data Granularity

### Time Ranges

> To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods:<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />

Reviewer: ```suggestion
To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods.<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
```

Reviewer: Removed  `:`

### Compliance Period

> You can also filter by compliance period to view your past activity and plan ahead:<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />

Reviewer: ```suggestion
You can also filter by compliance period to view your past activity and plan ahead.<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
```

Reviewer: @kimsauce: Is there any specific reason for using ":" here? I see it in the above sentence as well.

Author: Colon precedes an image, like this:
<img width="635" alt="Screenshot 2023-02-07 at 12 20 22 PM" src="https://user-images.githubusercontent.com/56411016/217356263-a92a21a7-0a75-40d3-aea2-a854b07c75b4.png">

Reviewer: That's a complete sentence and we can't put colon here. It's rule in grammar and the reason is colon and em dashes are always used to signal further clarification or used with lists.

Author: Per the example in my previous comment, this sentence (and the ones like it throughout the docs) are introducing images and code snippets. This formatting is used everywhere in tech docs (Google, Microsoft, GitHub, etc.). Can you please add to Docs Meeting agenda for discussion?

> ### Accessing Log Search

Reviewer 2: how about `Open In Log Search`

You can launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down into further granular details.

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> ### Accessing Log Search

Author: ```suggestion
### Open in Log Search
```

1. Go to **Manage Data** > **Monitoring** > **SLO**.
1. Click on any SLO line item.

> 1. Hover over the panel > click the three-dot kebab icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>

Reviewer: I'd rephrase this as:
```suggestion
1. Hover over the panel, click the three-dot kebab icon, and select the **Open in Log Search** option.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
```

Author: I'll add to next meeting agenda. Typically when introducing cascading UI elements, most tech docs use `>` (visually, it's easier to scan).
